kernel/packages/decentraland-amd/package.json

```diff
@@ -12,11 +12,17 @@
   "files": [
     "dist"
   ],
+  "prettier": {
+    "semi": true,
```

Comment:
```suggestion
    "semi": false,
```

Reply:
I did not want to change the styles that were already in place for the file. may that be another pr?

Reply:
then let's rather drop this section